_includes/02-image.md

```diff
@@ -0,0 +1 @@
+Replace this with an image, like your photo. Ensure you include some alt-text.
```

Comment:
![what is a github](https://user-images.githubusercontent.com/37124124/52881762-1f4cea80-312b-11e9-9663-134519df25d6.jpg)